specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/examples/GetACapacityReservation.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
please put this file in examples/compute folder

Reply:
updated